process/geometry/tfcoil_geometry.py

```diff
@@ -0,0 +1,191 @@
+"""
+Calculate radial and vertical coordinates for the geometry of the tf coils
+"""
+from typing import List, Tuple
+from process.geometry.geometry_parameterisations import RectangleGeometry
+from process.geometry.utils import ellips_fill
+
+
+def tfcoil_geometry_rectangular_shape(
+    x1: float,
+    x2: float,
+    x4: float,
+    x5: float,
+    y1: float,
+    y2: float,
+    y4: float,
+    y5: float,
+    tfcth: float,
+) -> List[RectangleGeometry]:
+    """Calculates rectangular geometries for tf coils in a picture frame/rectangular shape parametrization
+
+    :param x1: radial location of arc point 1
+    :type x1: float
+    :param x2: radial location of arc point 2
+    :type x2: float
+    :param x4: radial location of arc point 4
+    :type x4: float
+    :param x5: radial location of arc point 5
+    :type x5: float
+    :param y1: vertical location of arc point 1
+    :type y1: float
+    :param y2: vertical location of arc point 2
+    :type y2: float
+    :param y4: vertical location of arc point 4
+    :type y4: float
+    :param y5: vertical location of arc point 5
+    :type y5: float
+    :param tfcth: inboard tf coil thickness
+    :type tfcth: float
+    :return: list of RectangleGeometry - dataclass returning rectangular geometry parameters
+    :rtype: List[RectangleGeometry]
+    """
+    return_rects = []
+    return_rects.append(
+        RectangleGeometry(
+            center_x=x5 - tfcth,
+            center_z=y5 - tfcth,
+            width=tfcth,
+            height=(y1 - y5 + 2.0 * tfcth),
+        )
+    )
+    return_rects.append(
+        RectangleGeometry(
+            center_x=x4,
+            center_z=y4 - tfcth,
+            width=tfcth,
+            height=(y2 - y4 + 2.0 * tfcth),
+        )
+    )
+    return_rects.append(
+        RectangleGeometry(center_x=x5, center_z=y5 - tfcth, width=x4 - x5, height=tfcth)
+    )
+    return_rects.append(
```

Comment:
Again, what are all of these rectangles? A comment would be helpful to see what this function is doing